frontend/src/pages/ActivityReport/SectionOne.js

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Fieldset, Radio, Label, Dropdown, Textarea,
+} from '@trussworks/react-uswds';
+
+const grantees = [
+  'Grantee Name 1',
+  'Grantee Name 2',
+  'Grantee Name 3',
+];
+
+const nonGrantees = [
+  'CCDF / Child Care Administrator',
+  'Head Start Collaboration Office',
+  'QRIS System',
+  'Regional Head Start Association',
+  'Regional TTA/Other Specialists',
+  'State CCR&R',
+  'State Early Learning Standards',
+  'State Education System',
+  'State Health System',
+  'State Head Start Association',
+  'State Professional Development / Continuing Education',
+];
+
+const reasons = [
+  'reason 1',
+  'reason 2',
+];
+
+const otherUsers = [
+  'User 1',
+  'User 2',
+  'User 3',
+];
```

Comment:
For these kinds of consts, I'm used to having a directory or file that contains all the consts, and then both the code and the tests import those. Is that not common with FE or are we just thinking that because these will eventually pull from the database it's not really necessary?

Reply:
Since these constants will be replaced soon I didn't think separating them out was necessary